packages/api-explorer/package.json

```diff
@@ -108,8 +70,7 @@
     "react-markdown": "^4.3.1",
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
-    "styled-components": "^4.4.1",
-    "ts-jest": "^26.0.0"
+    "styled-components": "^4.4.1"
```

Comment:
As long as you're cleaning these up you should remove the duplicate types here, and probably move ace into dependendencies.